### DISCUSSION TITLE
К вопросу о майнере

### DISCUSSION THREAD

**Discussion by Klikker47** [Flowseal/zapret-discord-youtube#726] [General]
Я простой обыватель и вижу тут рефералку на майнер, а не переходы по серверам) чёт тут не так всё просто как кажется 
![Снимок](https://github.com/user-attachments/assets/6bdccc9c-f84a-41eb-b4c2-de468dbc6755)

**Comment by Flowseal** [comment]
Ссылка на донат разработчику WinDIvert
